pallets/registries/src/lib.rs

```diff
@@ -607,6 +609,7 @@ pub mod pallet {
 					revoked: false,
 					archived: false,
 					digest,
+					schema_id,
```

Comment:
@amarts Here do you see the need of storing the `schema_id` with the `registry` is associated with?

Reply:
Yes. It would be good to store.

Reply:
Ack.